ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp

```diff
@@ -62,14 +62,16 @@ class FoxgloveBridge : public rclcpp::Node {
     if (_useSimTime) {
       serverCapabilities.push_back(foxglove::CAPABILITY_TIME);
     }
+    const std::vector<std::string> supportedEncodings = {"cdr"};
```

Comment:
why doesn't this one use a static constant when the other one does?

Reply:
I guess because we originally planned to support both `cdr` and `idl`. Will change it in the future when we have some time for cleanups